chart/values.yaml

```diff
@@ -178,6 +178,10 @@ startupProbe:
   timeoutSeconds: 5
   periodSeconds: 10
   failureThreshold: 12
+
+# Additional taints to tolerate
+tolerations: {}
```

Comment:
Can we rename this to `additionalTolerations`?

After looking at this more I think that's better. Since it's more accurate to situation and there's nothing lost being more descriptive/specific here.

Reply:
Make sense, are you ok with extraTolerations?

Reply:
Yeah, that works for me. Thanks